app/pb_kits/playbook/pb_bar_graph/_bar_graph.html.erb

```diff
@@ -0,0 +1,17 @@
+<%= content_tag(:div, "",
+  id: object.id(object.id),
+  )
+%>
```

Comment:
For all Playbook Kits, the user needs to be able to add their own custom id, data-attributes and classes. To do this, we have to that the wrapping div work a specific way.

This should look like this:
```
<%= content_tag(:div, "",
    id: object.id("bar-graph"),
    data: object.data,
    class: object.classname("pb_bar_graph") %>
```

More details on this: https://github.com/powerhome/playbook/wiki/Understanding-Rails-Kit-HTML-Wrapper

Reply:
If you want the kit to require an ID from the user it should be written like this:

```
<%= content_tag(:div, "",
    id: object.id,
    data: object.data,
    class: object.classname("pb_bar_graph") %>
```